### ISSUE TITLE
Allow the back button in studies to be focusable

### ISSUE THREAD

**Issue by perclasson** [material-components/material-components-flutter-gallery#566]
Right now if you are inside of a study you cannot navigate to the back button with focus, as it is outside of the current navigator.

This issue is blocked by: https://github.com/flutter/flutter/issues/47344

migrated issue: https://github.com/flutter/gallery/issues/65
